src/Publisher/FilesystemPublisher.php

```diff
@@ -145,6 +181,10 @@ protected function publishPage($response, $url)
     {
         $success = true;
         if ($path = $this->URLtoPath($url)) {
+            // little hack to make sure we do not include pages with live forms.
+            if (stripos($response->getBody(), 'name="SecurityID"')) {
```

Comment:
This feels a bit out of scope and is a very fragile implementation because `SecurityID` is configurable. I'd prefer this to be removed from this PR

Reply:
made it optional - I still think this is a very easy way to avoid pages with forms.